### PR TITLE
Make filename configurable for hokusai/test

### PR DESCRIPTION
**Pull request by zephraph**
I'm working on a metaphysics change that requires running a set of tests in isolation via hokusai. I want to be able to have a separate command just for that.

### CODE REVIEW

**Comment by zephraph** [comment]
artsy/hokusai@dev:9d4e8a1dc214b0e73f119df57a29103b

**Comment by zephraph** [comment]
I'm experimenting with this in https://github.com/artsy/metaphysics/pull/1846/commits/283789497f9f81136377deba3b7ed5c501e3e7ac

File: src/hokusai/hokusai.yml [hunk 1]
```diff
@@ -55,11 +55,16 @@ commands:
...
+      filename:
+        type: string
+        default: ./hokusai/test.yml
+        description: The docker-compose yaml file to use
     steps:
       - setup-docker
       - run:
           name: Test
-          command: hokusai test
+          command: hokusai test -f << parameters.filename >>
```
Comment (eessex) [inline src/hokusai/hokusai.yml]: What happens if you pass nothing?

Comment (zephraph) [reply src/hokusai/hokusai.yml]: It uses the default as defined by the parameter block above. That default is the same as Hokusai’s.

**Pull request merged**